doc/en/user/source/styling/sld/extensions/pointsymbols.rst

```diff
@@ -286,7 +286,75 @@ For example:
       </Graphic>
     </PointSymbolizer>
 
-In this example an SVG graphic is being used, so the size is specified explicitly. 
+In this example an SVG graphic is being used, so the size is specified explicitly.
+
+Parameterized (like QGIS) SVG images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Just like QGIS, Geoserver can handle SVG images in which parts of the SVG-attributes are 
+named parameters as defined in the `SVG Parameters 1.0 spec <https://www.w3.org/TR/SVGParamPrimer/>`_.
```

Comment:
I did not know this was part of the SVG specification cool :)

* `GeoServer`

* I try and use links of the form:
  ```
  `SVG Parameters 1.0 spec <https://www.w3.org/TR/SVGParamPrimer/>`__
  ```
  
  The two under scores are processed differently by sphinx and avoids a conflict if the same URL is referenced twice in the docs.